NEWS.rst

```diff
@@ -1,6 +1,13 @@
 Version History
 ===============
 
+
+Version 0.6.0 (2020-12-01):
```

Comment:
You should check the generated HTML file, which is broken (https://sfs-python--167.org.readthedocs.build/en/167/version-history.html).

The bullet points have to be indented, like they are below.